install.sh

```diff
@@ -359,6 +359,17 @@ stage_local_airgap_tarball() {
         cp -f "${INSTALL_RKE2_ARTIFACT_PATH}/rke2-images.${SUFFIX}.tar.gz" "${TMP_AIRGAP_TARBALL}"
         AIRGAP_TARBALL_FORMAT=gz
     fi
+    # Search for each CNI we support and copy it to the airgap directory
+    CNIS="calico canal cilium flannel"
```

Comment:
I don't love having the CNI list hardcoded in the install script. Any reason we shouldn't just move or copy `rke2-images.*`?

Reply:
also, the path ${INSTALL_RKE2_AGENT_IMAGES_DIR} is available here? IIRC it's created on the install phase (after this line446)

Reply:
and if you're doing a copy as Brandond suggest the lines 636 and 637 (verify_airgap_tarball and install_airgap_tarball are neeeded ?